language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -32,9 +32,7 @@ import Data.Maybe
 import Data.Bifoldable
 import Options.Applicative
 import System.Directory
-import System.FilePath hiding ((<.>), (</>))
-import System.FilePath.Posix((</>)) -- Make sure we generate / on all platforms.
```

Comment:
We don't want `/` on all platforms in filenames. We don't use `</>` to calculate paths in import statements.

Reply:
Ok